templates/repo/issue/view_content/pull.tmpl

```diff
@@ -131,38 +131,37 @@
 		{{$showGeneralMergeForm := false}}
 		<div class="ui attached merge-section segment {{if not $.LatestCommitStatus}}no-header{{end}}">
 			{{if .Issue.PullRequest.HasMerged}}
-				<div class="item text">
-					{{if .Issue.PullRequest.MergedCommitID}}
-						{{$link := printf "%s/commit/%s" $.Repository.Link (.Issue.PullRequest.MergedCommitID|PathEscape)}}
-						{{if eq $.Issue.PullRequest.Status 3}}
-							{{$.locale.Tr "repo.pulls.manually_merged_as" ($link|Escape) (ShortSha .Issue.PullRequest.MergedCommitID) | Safe}}
```

Comment:
I think we need to keep this.

Manual merge is a very special case. Have you tried it? I guess users need to know the commit ID for a manually merged PR.

Reply:
But isn't the commit hash the same as in the "merged" timeline entry?

The message difference with the "manually merged" instead of "merged", I can add back I guess.

Reply:
Does anyone have gitea.com/try.gitea.io example of such a manual merge?

Reply:
Do you have a example to look at at one of the sites as well?

Reply:
Sorry I didn't get the point, do you mean "to see  a screenshot what the UI looks like" or "to visit a page with that UI" ?

Reply:
I managed to reproduce until step 3, but can not execute step 4 "Click the ‘Manually merged’ button". Why is there no button? This is what I see:

<img width="861" alt="Screenshot 2023-04-23 at 19 01 54" src="https://user-images.githubusercontent.com/115237/233853842-13d43b16-14f4-4a4b-bb84-a355b01264bc.png">

Reply:
It appears `$.StillCanManualMerge` is false so the button is hidden.

Reply:
On my try.gitea.io , some of my PRs have this:


![image](https://user-images.githubusercontent.com/2114189/233853950-5d615ec9-8ead-4b5f-b372-18cf1243c9e5.png)

Reply:
Oh, you also need this ..... so we really need some comments .......


![image](https://user-images.githubusercontent.com/2114189/233854020-12134817-1d6d-4183-8704-31b76b74f4d4.png)

Reply:
Yes, that did the trick. So this screenshot confirms that the commit hash is duplicate with the timeline entry, and does not need to be included in the message to avoid useless info. Would you agree?

<img width="864" alt="Screenshot 2023-04-23 at 19 09 31" src="https://user-images.githubusercontent.com/115237/233854248-048e544f-f86a-4f18-bed8-a1c1f3fd2c55.png">

Reply:
Thank you for the confirmation.

Reply:
And here is the same view after this PR. All info is still there, the ""manually" word is in the timeline. So I'd say we keep it like this.

<img width="865" alt="Screenshot 2023-04-23 at 19 13 42" src="https://user-images.githubusercontent.com/115237/233854435-a22c1569-6b38-4d3c-b076-ec78a320356a.png">